ensime_shared/editor.py

```diff
@@ -278,8 +278,8 @@ def to_quickfix_item(self, file_name, line_number, message, tpe):
                 "text": message,
                 "type": tpe}
 
-    def write_quickfix_list(self, qflist):
-        self._vim.command('call setqflist({!s})'.format(qflist))
+    def write_quickfix_list(self, qflist, title):
+        self._vim.command("call setqflist({!s}, 'r', 'Ensime - {}')".format(qflist, title))
```

Comment:
Oh my, really good catch here, dropping Vim 7 support wasn't my intention at all :smile:. I just naively followed nvim docs for the function trusting on them being compatible across vim versions/flavors.

- [ ] Will rework this to make sure that ir runs everywhere (aka add a bunch of `if`s :disappointed:)